src/drivers/display/renesas_glcdc/lv_renesas_glcdc.c

```diff
@@ -76,6 +77,9 @@ static glcdc_runtime_cfg_t  g_layer_change;
 display_t g_display0_cfg;
 #endif /*_RENESAS_RX_*/
 
+static void * rotation_buffer = nullptr;
```

Comment:
Will this compile with an error? `nullptr` is used in C++11, while C uses `NULL`.